docs/dev/bak/search.json

```diff
@@ -0,0 +1,113 @@
+[
```

Comment:
I assume this is the actual search content? And we'll have to have the generator create this?

Reply:
Yes, this is real search content. Initially, I am glad to create a record for each RDP page in the search index. It doesn't take long. This will get us started. We don't need to update the search index every time someone submits a PR with a few changes, so, if we do hook the search-index-generation process into the generator, which I think is a good idea, then I think kicking off the process needs to be conditional in some way, either by a human-set command-line flag, or by some programmatic condition that I can't think of.